edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudProxy.cs

```diff
@@ -21,6 +22,15 @@ namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy
 
     class CloudProxy : ICloudProxy
     {
+        [Flags]
+        enum Subscriptions
```

Comment:
Subscriptions enum and the add/remove operations could be encapsulated in a class

Reply:
moved logical operation into a class